docs/source-pytorch/fabric/fabric.rst

```diff
@@ -164,40 +164,41 @@ Furthermore, you can access the current device from ``fabric.device`` or rely on
 
 ----------
 
+*******************
+Fabric in Notebooks
+*******************
 
-Distributed Training Pitfalls
-=============================
 
-The :class:`~lightning_fabric.fabric.Fabric` provides you with the tools to scale your training, but there are several major challenges ahead of you now:
+Fabric works exactly the same way in notebooks (Jupyter, Google Colab, Kaggle, etc.) if you only run in a single process or a single GPU.
+If you want to use multiprocessing, for example multi-GPU, you can put your code in a function and pass that function to the
+:meth:`~lightning_fabric.fabric.Fabric.launch` method:
 
 
-.. list-table::
-   :widths: 50 50
-   :header-rows: 0
+.. code-block:: python
+
+
+    # Notebook Cell
+    def train(fabric):
+
+        model = ...
+        optimizer = ...
+        model, optimizer = fabric.setup(model, optimizer)
+        ...
+
 
-   * - Processes divergence
-     - This happens when processes execute a different section of the code due to different if/else conditions, race conditions on existing files and so on, resulting in hanging.
-   * - Cross processes reduction
-     - Miscalculated metrics or gradients due to errors in their reduction.
-   * - Large sharded models
-     - Instantiation, materialization and state management of large models.
-   * - Rank 0 only actions
-     - Logging, profiling, and so on.
-   * - Checkpointing / Early stopping / Callbacks / Logging
-     - Ability to customize your training behavior easily and make it stateful.
-   * - Fault-tolerant training
-     - Ability to resume from a failure as if it never happened.
+    # Notebook Cell
+    fabric = Fabric(accelerator="cuda", devices=2)
+    fabric.launch(train)  # Launches the `train` function on two GPUs
 
 
-If you are facing one of those challenges, then you are already meeting the limit of :class:`~lightning_fabric.fabric.Fabric`.
-We recommend you to convert to :doc:`Lightning <../starter/introduction>`, so you never have to worry about those.
+As you can see, this function accepts one argument, the ``Fabric`` object, and it gets launched on as many devices as specified.
 
 
 ----------
 
-************
-Fabric Flags
-************
+********************
+Fabric Configuration
```

Comment:
Why change to configuration? I think "flags" is how we call it in trainer too, right?

Reply:
Reverted back to "Flags". But it's not technically the right term. Most Arguments aren't flags.